pkg/actors/internal/placement.go

```diff
@@ -355,7 +355,7 @@ func (p *ActorPlacement) establishStreamConn() (v1pb.Placement_ReportDaprStatusC
 		conn, err := grpc.Dial(serverAddr, opts...)
 	NEXT_SERVER:
 		if err != nil {
-			log.Debugf("error connecting to placement service: %v", err)
+			log.Warnf("error connecting to placement service: %v", err)
```

Comment:
Could you tell me the reason of this change?

Reply:
It's for the issue
https://github.com/dapr/dapr/issues/2896

When the placement service cannot be connected, maybe we should log a warning.

Reply:
I agree. There will be a new change to ignore placement if not endpoint is set for placement service.